packages/wrangler/src/cfetch/index.ts

```diff
@@ -106,5 +106,6 @@ function throwFetchError(
 }
 
 function hasCursor(result_info: unknown): result_info is { cursor: string } {
-  return (result_info as { cursor: string } | undefined)?.cursor !== undefined;
+  const cursor = (result_info as { cursor: string } | undefined)?.cursor;
+  return cursor !== undefined && cursor !== "";
```

Comment:
This change is better fixed in #335 - perhaps remove it from here, or rebase on that PR's branch?

Reply:
Yeah I just left it here so I could test it before 335 landed and so I could write tests. Writing tests right now.